onnxruntime/core/providers/dnnl/dnnl_execution_provider.cc

```diff
@@ -5,42 +5,39 @@
 #pragma warning(disable : 4996)
 #endif
 
-#include "core/framework/allocator.h"
-#include "core/framework/compute_capability.h"
-#include "core/framework/kernel_registry.h"
-#include "core/providers/dnnl/subgraph/dnnl_func_kernel.h"
+#include "core/providers/shared_library/provider_author.h"
+#include <unordered_set>
+#include "subgraph/dnnl_func_kernel.h"
 #include "dnnl_execution_provider.h"
 #include "dnnl_fwd.h"
 
-namespace {
+namespace onnxruntime {
```

Comment:
why in onnxruntime namespace now when previously was anon.
seems specific to this file scope.

Reply:
I'm not sure why I changed that. I switched it back to the anonymous namespace